packages/plugin-workspace-tools/sources/commands/foreach.ts

```diff
@@ -77,7 +79,7 @@ export default (clipanion: any, pluginConfiguration: PluginConfiguration) => cli
   }))
```

Comment:
Just noticed that we don't have examples for this command - can you add some? I think two good ones would be "publish everything --tolerate-republish" and "run build on everything".